spec/mentions_spec.rb

```diff
@@ -95,4 +95,40 @@ def para(content)
       expect(basic_post.output).to start_with(para(result.sub(default_src, mentions_src)))
     end
   end
+
+  context "with the SSL and GITHUB_HOSTNAME environment variables set" do
+    let(:ssl)                  { "true" }
+    let(:github_hostname)      { "github.vm" }
+    let(:default_mention_base) { "https://github.vm" }
+
+    before(:each) do
+      ENV["SSL"] = ssl
+      ENV["GITHUB_HOSTNAME"] = github_hostname
+    end
+
+    after(:each) do
+      ENV.delete("SSL")
+      ENV.delete("GITHUB_HOSTNAME")
+    end
+
+    it "has a default source based on SSL and GITHUB_HOSTNAME" do
+      expect(mentions.mention_base).to eql(default_mention_base)
+    end
+
+    it "uses correct mention URLs when SSL and GITHUB_HOSTNAME are set" do
+      # Re-render the site, so that ENV is used
+      site.render
```

Comment:
This is done because `ENV` values are set in this context's `before(:each)` block, but the site is rendered in the parent `before(:each)` block, meaning the original render doesn't get the `ENV` values.